src/main/java/org/systemli/keycloak/Constants.java

```diff
@@ -0,0 +1,10 @@
+package org.systemli.keycloak;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class Constants {
+        public String REALM_DOMAIN = "userliRealmDomain";
+        public String BASE_URL = "userliBaseUrl";
+        public String KEYCLOAK_API_TOKEN = "userliKeycloakToken";
```

Comment:
```suggestion
        public final String REALM_DOMAIN = "userliRealmDomain";
        public final String BASE_URL = "userliBaseUrl";
        public final String KEYCLOAK_API_TOKEN = "userliKeycloakToken";
```

Constants should be final to ensure they cannot be modified.